### PR TITLE
[Minor Update] Clarify in cleanlab-studio readme (+ banner) what python v supported

### PR DESCRIPTION
**Pull request by ulya-tkch**

### CODE REVIEW

**Review by ulya-tkch — COMMENTED** [review]
add clarification to readme.

File: README.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ If you already have the client installed and wish to upgrade to the latest versi
 pip install --upgrade cleanlab-studio
 ```
 
+Cleanlab Studio API client requires Python >= 3.8.
```
Comment (jwmueller) [inline README.md:25]:
```suggestion
```

Comment (jwmueller) [reply README.md:25]: think the banner already conveys this, so omitting for now

**Review by jwmueller — APPROVED** [review]
thanks!

**Pull request merged**